src/Void/Init.fs

```diff
@@ -5,6 +5,23 @@ open Void.Lang.Interpreter
 open Void.Lang.Editor
 open Void.ViewModel
 
+type VoidOptions = {
+    EnableVerboseMessageLogging : bool
+}
+
+module Options =

```

Comment:
Also, long-term we may want to call these settings, because F# has `Option` instead of `Maybe`.